### PR TITLE
Newgitbranch

### PR DESCRIPTION
**Pull request by galogan**
1st comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # assign2git
 Grant Logan
 First comment
+testbranch change
+new git branch on GitHub
```
Comment (galogan) [inline README.md:5]: i agree with the changes being made

Comment (galogan) [reply README.md:5]: changes were made correctly

**Review by btitus1 — APPROVED** [review]
Looks good

**Review by btitus1 — APPROVED** [review]
Approved

**Review by btitus1 — APPROVED** [review]
Approved...Again

**Pull request closed without merging**